### PR TITLE
Feature/rebrand

### PR DESCRIPTION
**Pull request by tolyo**
Initial namespace migration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,19 +33,19 @@
...
     "gulp-load-plugins": "^0.8.0",
     "gulp-markdown": "^1.0.0",
     "gulp-minify-css": "^1.2.2",
     "gulp-ng-html2js": "^0.1.8",
     "gulp-prompt": "^0.1.2",
     "gulp-rename": "^1.2.0",
     "gulp-rsync": "^0.0.4",
-    "gulp-ruby-sass": "1.0.0-alpha",
-    "gulp-sass": "^1.3.3",
+    "gulp-ruby-sass": "^2.0.6",
```
Comment (soumak77) [inline package.json:43]: I think this package can be removed.  It was needed in earlier versions, but now `gulp-sass` should be the only sass dependency.

Comment (AntJanus) [inline package.json:43]: good point!

File: js/angular/components/actionsheet/actionsheet.js [hunk 2]
```diff
@@ -9,6 +9,14 @@
     .service('FoundationActionSheet', FoundationActionSheet)
   ;
 
+  angular.module('base.actionsheet', ['foundation.core'])
```
Comment (soumak77) [inline js/angular/components/actionsheet/actionsheet.js]: module should depend on `base.core`, not `foundaton.core`

**Comment by soumak77** [comment]
Inside `deploy:dist` gulp task, rename `foundation-apps.*` distro files to `base-apps.*`

File: package.json (hunk 1 above)
Comment (tolyo) [inline package.json:43]: I believe its being used in `test:sass` line 255 gulp.

Comment (soumak77) [inline package.json:43]: I see that now.  We should be using the same sass library to test with, so that should be changed to use `gulp-sass`.

Comment (soumak77) [inline package.json:43]: You don't need to make that change with this PR if you don't want to.  I've created an issue to address it https://github.com/base-apps/angular-base/issues/6.

Comment (tolyo) [inline package.json:43]: Yeah. Let's do a minimal viable release for now.

Comment (soumak77) [inline package.json:43]: I think it's worth merging this change (https://github.com/base-apps/angular-base/pull/8).  It's straight forward and removes an obsolete dependency.

Comment (tolyo) [inline package.json:43]: Roger

**Comment by tolyo** [comment]
I think we ll need two distros for now as $templateCache is being populated with respect to angular modules - `foundation` and `base` in our case.

**Comment by soumak77** [comment]
Agreed.  We will support the `foundation` naming for modules and files for all 1.2.\* releases.  Starting with 1.3 we can remove references to foundation entirely.

**Pull request merged**